### PR TITLE
[5.6] Update Cookie docblocks

### PR DESCRIPTION
**Pull request by mnabialek**

### CODE REVIEW

File: src/Illuminate/Contracts/Cookie/Factory.php [hunk 1]
```diff
@@ -10,8 +10,8 @@
      * @param  string  $name
      * @param  string  $value
      * @param  int     $minutes
-     * @param  string  $path
-     * @param  string  $domain
+     * @param  string|null  $path
```
Comment (JackPriceBurns) [inline src/Illuminate/Contracts/Cookie/Factory.php]: Shouldn't these be aligned? :)

File: src/Illuminate/Contracts/Cookie/Factory.php [hunk 2]
```diff
@@ -25,8 +25,8 @@ public function make($name, $value, $minutes = 0, $path = null, $domain = null,
      *
      * @param  string  $name
      * @param  string  $value
-     * @param  string  $path
-     * @param  string  $domain
+     * @param  string|null  $path
```
Comment (JackPriceBurns) [inline src/Illuminate/Contracts/Cookie/Factory.php]: Shouldn't these be aligned? :)

File: src/Illuminate/Foundation/helpers.php [hunk 3]
```diff
@@ -299,11 +299,11 @@ function config_path($path = '')
     /**
      * Create a new cookie instance.
      *
-     * @param  string  $name
-     * @param  string  $value
+     * @param  string|null  $name
```
Comment (JackPriceBurns) [inline src/Illuminate/Foundation/helpers.php]: Shouldn't these be aligned? :)

File: src/Illuminate/Contracts/Cookie/Factory.php [hunk 4]
```diff
@@ -39,8 +39,8 @@ public function forever($name, $value, $path = null, $domain = null, $secure = f
      * Expire the given cookie.
      *
      * @param  string  $name
-     * @param  string  $path
-     * @param  string  $domain
+     * @param  string|null  $path
```
Comment (JackPriceBurns) [inline src/Illuminate/Contracts/Cookie/Factory.php]: Shouldn't these be aligned? :)

**Review by JackPriceBurns — CHANGES_REQUESTED** [review]

File: src/Illuminate/Contracts/Cookie/Factory.php (hunk 1 above)
Comment (mnabialek) [reply src/Illuminate/Contracts/Cookie/Factory.php]: Others parameters in this and others methods are not aligned so I won't align those if not really necessary.

**Comment by sisve** [comment]
While these are changes in docblock types, they are only changing the docblocks to match the default values (null) in the signatures. As such this seems like a non-breaking change, and implementation will already support null

**Comment by taylorotwell** [comment]
Is this for some kind of analysis tool?

**Pull request closed without merging**